incubator/openfaas/values.yaml

```diff
@@ -0,0 +1,52 @@
+functionNamespace:
+async: false
+armhf: false
+exposeServices: true
+serviceType: NodePort
+rbac: true
+
+# images
+images:
+  controller: functions/faas-netesd:0.3.4
+  controllerArmhf: functions/faas-netesd-armhf
```

Comment:
For images the version and pull policy should be separate from the location. You can read about that in the [review guidelines](https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md#configuration)